stl/inc/random

```diff
@@ -752,16 +756,11 @@ public:
 
 protected:
     template <class _Gen>
-    void _Seed(_Gen& _Gx, bool _Readcy, true_type) { // reset sequence from numeric value
-        linear_congruential<_Seed_t, 40014U, 0U, 2147483563U> _Lc(_Gx == 0U ? default_seed : _Gx);
+    void _Seed(_Gen& _Gx, bool _Readcy) { // reset sequence from numeric value
+        linear_congruential<_Seed_t, 40014U, 0U, 2147483563U> _Lc{_Gx == 0U ? default_seed : _Gx};

```

Comment:
For the record - here is the case that failed (it required the non-Standard engine, because the Standard one has `seed()` overloads that avoid the problem):

```
C:\Temp>type meow.cpp
```
```cpp
#include <random>
using namespace std;

int main() {
    subtract_with_carry<unsigned int, 24, 10, 24> engine{};
    unsigned long long ull = 1729;
    engine.seed(ull);
}
```
```
C:\Temp>cl /EHsc /nologo /W4 meow.cpp
meow.cpp
D:\GitHub\STL\out\build\x64\out\inc\random(760): error C2398: Element '1': conversion from '_Gen' to '_Uint' requires a narrowing conversion
        with
        [
            _Gen=unsigned __int64
        ]
        and
        [
            _Uint=unsigned int
        ]
```